lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1185,6 +1185,69 @@ static void helperSetter(Constant *constT, std::vector<datatype> &vec) {
   }
 }
 
+template <typename T, typename datatype>
+static void helperSetter(const glow::Constant *constT,
+                         std::vector<datatype> &vec) {
+  auto constH = constT->getPayload().getHandle<T>();
+  for (size_t i = 0; i < constH.size(); ++i) {
+    vec.push_back(constH.at({i}));
+  }
+}
```

Comment:
We don't need this, do we? It's not called anywhere it seems.

Reply:
Have removed it.